src/jail_run.c

```diff
@@ -271,6 +273,8 @@ jail_run(
         opt = g_strdup_printf(FIREJAIL_DBUS_LOG_OPT_FMT, trace_dir);
         g_ptr_array_add(args, opt);
         g_ptr_array_add(args_alloc, opt);
+        if (!dbus_system_filter_added)
```

Comment:
Shouldn't this be the other way around? I.e.
```
if (!dbus_system_filter_added) {
    g_ptr_array_add(args, (gpointer) FIREJAIL_DBUS_SYSTEM_LOG);
    g_ptr_array_add(args, (gpointer) FIREJAIL_DBUS_USER_LOG);
}
```

Reply:
The clode block above adds that system log if there is system bus in use, no need to add that twice? That was the main cause I think.

Reply:
:thinking:

Reply:
Okay, so... why is this needed? It's not obvious from the code as it does already add both FIREJAIL_DBUS_SYSTEM_FILTER and FIREJAIL_DBUS_SYSTEM_LOG arguments on lines 254 and 257 respectively which makes me think that line 278 is redundant and could be removed.

Reply:
Ah, or maybe now I understand why this is needed: When tracing is used we must filter both buses so that we can catch when the bus is used. It doesn't really have much anything to do with whether we have allowed access to something in the bus.